cmd/akash/session/session.go

```diff
@@ -198,7 +207,7 @@ func (ctx *session) Key() (keys.Info, error) {
 }
 
 func (ctx *session) Password() (string, error) {
-	return constants.Password, nil
+	return viper.GetString("PASSWORD"), nil
```

Comment:
use `flagPassword` constant - both uppercase and lowercase has been used for this.